sdk/ai/azure-ai-agents/README.md

```diff
@@ -320,22 +320,20 @@ Here is an example to integrate Azure AI Search:
 <!-- SNIPPET:sample_agents_azure_ai_search.create_agent_with_azure_ai_search_tool -->
 
 ```python
-conn_id = os.environ["AI_AZURE_AI_CONNECTION_ID"]
-
-print(conn_id)
-
-# Initialize agent AI search tool and add the search index connection id
-ai_search = AzureAISearchTool(
-    index_connection_id=conn_id, index_name="sample_index", query_type=AzureAISearchQueryType.SIMPLE, top_k=3, filter=""
-)
-
-# Create agent with AI search tool and process agent run
-project_client = AIProjectClient(
+with AIProjectClient(
     endpoint=os.environ["PROJECT_ENDPOINT"],
     credential=DefaultAzureCredential(),
-)
-
-with project_client:
+  ) as project_client:
+    conn_id = project_client.connections.get_default(ConnectionType.AZURE_AI_SEARCH).id
```

Comment:
how the default is defined, is there possibility e.g. have multiple azure ai search connections in project and where one if default?

Reply:
In our implementation we [are listing connections of a given type and select the first one](https://github.com/Azure/azure-sdk-for-python/blob/60e06f27a851ef1f18ddb1c6fc06fd800953e7c6/sdk/ai/azure-ai-projects/azure/ai/projects/operations/_patch_connections.py#L63C13-L63C19).